isdown/README.md

```diff
@@ -0,0 +1,52 @@
+# Agent Check: IsDown
+
+## Overview
+
+[IsDown][1] is a status page aggregator and outage monitoring tool that helps businesses monitor their dependencies. You can provide your team real-time monitoring and instant notifications for outages in all your tools and cloud providers. IsDown monitors more than 2000 status pages.
+
+With this integration, you can receive alerts from third-party dependencies in Datadog, monitor business critical services, and understand the frequency of outages all within the out-of-the-box dashboard.
+
+## Setup
+
+### IsDown
+
+To connect Datadog to IsDown, you need to generate an [API key][5] in Datadog and add it to your IsDown account.
+
+1. Use your existing account or create a new one in [IsDown][1].
+2. Log in to your account and go to the **Notifications** page.
+3. Add your API Key and select if you're using the US or EU [Datadog site][4].
+4. Click **Save**.
+5. Select the services you want to monitor.
+6. In each service you can choose the notification settings.
+
+
+### Uninstallation
+
+1. Go to the **Notifications** page in IsDown.
+2. Unselect Datadog and click **Save**.
+3. In Datadog, delete your API Key dedicated to IsDown.
```

Comment:
This should be updated if your integration is using OAuth.

Reply:
Hi @eho1307. I think this stays the same, or I'm supposed to delete the API Key from the application side?

Reply:
You can instead say "Ensure that all API keys associated with this integration have been disabled by searching for the integration name on the [API Keys management page](/organization-settings/api-keys?filter=) in Datadog."

Reply:
Sounds good! Thank you.